modules/block-interactions/src/main/java/unnamed/mmo/blocks/handlers/CropHandler.java

```diff
@@ -0,0 +1,82 @@
+package unnamed.mmo.blocks.handlers;
+
+import net.minestom.server.entity.Entity;
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.entity.metadata.item.ItemEntityMeta;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.instance.block.BlockHandler;
+import net.minestom.server.item.ItemStack;
+import net.minestom.server.utils.NamespaceID;
+import org.jetbrains.annotations.NotNull;
+import unnamed.mmo.blocks.BlockInteractionUtils;
+
+public class CropHandler implements BlockHandler {
+
+    private final CropBlockData blockData;
```

Comment:
I'm not a fan of stateful block handlers if it can be avoided, tags on the block itself are typically more convenient to save.

Reply:
Good point, I'll work on storing the CropBlockData as tags on the block itself and read from that instead.